src/Shell/Task/MigrationTask.php

```diff
@@ -60,7 +60,7 @@ public function templateData()
 
         $action = $this->detectAction($className);
 
-        if ($action === null) {
+        if (empty($action)) {
```

Comment:
See the signature of that method. It now returns an empty array instead of two return types which is usually a bad thing and a code smell. Instead it should always return array.